mas-storage/build.gradle

```diff
@@ -100,74 +100,101 @@ task javadoc(type: Javadoc) {
     include '**/*MAS*.java'
 }
 
-//jcenter integration
 group 'com.ca'
```

Comment:
is "group" var still used by anything? it's unclear the actual effect of
`group` vs.
`PUBLISH_GROUP_ID` (in gradle.properties) vs.
`groupId`